uclapi.openapi.json

```diff
@@ -4244,6 +4810,1679 @@
           }
         }
       }
+    },
+    "/libcal/space/locations": {
+      "get": {
+        "summary": "Gets all LibCal locations",
+        "tags": [ "LibCal" ],
```

Comment:
I think you might have forgotten to actually add this tag to the `tags` array (near the top of the file)!